service_container.module

```diff
@@ -8,28 +8,36 @@
 // Core Hooks
 
 /**
+ * Implements hook_stream_wrappers_alter().
+ */
+function service_container_stream_wrappers_alter(&$wrappers) {
+  if (class_exists('ServiceContainer')) {
+    ServiceContainer::init();
+  }
+}
+
+/**
+ * Implements hook_enable().
+ */
+function service_container_enable() {
+  if (class_exists('ServiceContainer')) {
+    ServiceContainer::reset();
```

Comment:
hook_enable() only runs for the module being enabled, don't we want to run this for all modules and hence use hook_modules_enabled() instead?